docs/api/innrapportering-skattefrieutbetalinger.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Innrapportering Skattefrie utbetalinger fra offentlige myndigheter API
+slug: /api/innrapportering-skattefrieutbetalinger
+folder: api
+sidebar: mydoc_sidebar
+datatable: true
+tags: [ API, skattefrieutbetalinger, vitenskapelig, forskning, yrkesopplæring ]
+keywords: [ grunnlagsdata ]
+last_updated: Mar 03, 2025
+hide_table_of_contents: true
+---
+
+<Summary>Tjeneste for innrapportering Skattefrie utbetalinger fra offentlige myndigheter (RF-1319)</Summary>
+
+<Tabs underline={true}>
+<TabItem headerText="Om tjenesten" itemKey="itemKey-1" default>
+
+For generell informasjon om tjenestene se egne sider om:
+
+* [Bruk av tjenestene](../om/bruk.md)
+* [Sikkerhetsmekansimer](../om/sikkerhet.md)
+* [Systemtilgang](../om/systemtilgang.md)
+* [Feilhåndtering](../om/feil.md)
+* [Versjonering](../om/versjoner.md)
+* [Teknisk spesifikasjon](../om/tekniskspesifikasjon.md)
+
+## Scope
+
+Følgende scope skal benyttes ved autentisering i Maskinporten: `skatteetaten:innrapporteringskattefrieutbetalinger`
+
+## Delegering
+
+Tilgang til dette API-et kan delegeres i Altinn, f.eks. dersom leverandør benyttes for den tekniske oppkoblingen.
+
+## Systemtilgang
+
+Bruk av API-et krever systemtilgang, som er ny funksjonalitet i Maskinporten levert av Digdir. 
+Informasjon vedr. dette finnes [her](../om/systemtilgang.md). 
+
+For å kunne benytte dette api'et med systemtilgang må man gi følgende rettighet til systemet ved opprettelse i systemregisteret:
+```JSON
+"Rights": [
+    {
+      "Resource": [
+        {
+          "value": "ske-innrapportering-skattefrie-utbetalinger",
+          "id": "urn:altinn:resource"
+        }
+      ]
+    }
+  ]
+```
+
+## Teknisk spesifikasjon
+
+URL-er til API-et, beskrivelsen av parameterne, endepunkter og respons ligger i Open API spesifikasjonen på
+[SwaggerHub](https://app.swaggerhub.com/apis/skatteetaten/innrapportering-skattefrie-utbetalinger-api/0.0.1)
+
+Nødvendige åpninger i en evt. brannmur er beskrevet [her](../om/sikkerhet.md)
+
+API-et for innrapportering av skattefrie utbetalinger fra offentlige myndigheter har to endepunkter
+
+* __POST innsending__: Mottar tredjepartsopplysninger for skattefrie utbetalinger fra offentlige myndigheter. Ett kall mot API-et er en rapportering for en organisasjon gitt av en oppgavegiver og som gjelder et inntektsår.
```

Comment:
Er det riktig at det for innsending står "Mottar tredjepartsopplysninger..."? Det høres litt bakvendt ut, denne dokumentasjonen er vel for innsender (som sender inn tredjepartsopplysninger).